### ISSUE TITLE
Adding our logo to show in emails from newsletter@ourworldindata.org

### ISSUE THREAD

**Issue by CGiattino** [owid/owid-grapher#3105]
## Description

We'd like for our logo to show in the newsletter emails that we send to subscribers — but currently this is not possible. For more context see [this Slack thread](https://owid.slack.com/archives/C023SHCDB1Q/p1702281600528129) between Charlie, Matthieu, and Lars.

## Expected behaviour

Our logo should show in emails sent from newsletter@ourworldindata.org.

**Comment by mlbrgl** [comment]
@danyx23 @ikesau triaged

**Comment by ikesau** [comment]
Looks like Mailchimp/Gmail have updated on this recently: https://www.mailgun.com/blog/email/google-bimi-update-2024/

**Comment by CGiattino** [comment]
thanks @ikesau! so sounds like to get our logo in email inboxes, we'd need either a Verified Mark Certificate (VMC) or Common Mark Certificate (CMC).

The VMC route seems quite hard and unlikely for us: it requires that "your logo be trademarked with an intellectual property office that’s recognized by VMC issuers. We recommend working with your legal team or a lawyer to get your logo trademarked. The trademark process can take 6 to 12 months."

The CMC route might be possible though — it would be a bit of paperwork and a video call. According to the DigiCert website, which I believe would be providing the mark:

> These steps include:
> 
> Confirm your logo has been used for at least a year or is similar to your registered trademark.
> 
> Provide notarized copies of personal identity documents to confirm the identity of the person at your organization applying for the CMC or conduct a live webcam interview with a member of DigiCert's validation team to validate the person's identity requesting the CMC.
> 
> Complete an in-person or video call meeting with a DigiCert validation member to confirm your identity matches those documents.

I can follow up on Slack with Este about this

**Comment by eoo-owid** [comment]
Thanks, the logo isn't trademarked, but CMC should be fine, yep. I think it's worth trying this. Not sure who needs to apply but it could be you or Valerie if needed.

(We do plan to trademark the logo and name but this isn't going to happen very soon, so I would just not count on it at all.)

**Comment by CGiattino** [comment]
@eoo-owid After looking into this a bit more, I see that DigiCert (which appears to be our only option) certificates [cost $99/month](https://www.digicert.com/tls-ssl/compare-mark-certificates) — I think this certificate seems like more of a "nice to have" than a necessity that would make a big difference, so I'm thinking this would not be worth paying for. Shout if you have other thoughts.

**Comment by eoo-owid** [comment]
I agree, if the only benefit is a tiny logo on emails, it's not worth paying this much. I'd just leave it (and if/when we register trademarks we can reconsider the other option)
